public/locales/en/translation.json

```diff
@@ -0,0 +1,264 @@
+{
+  "c": {
```

Comment:
What does "c" mean? Can you put a bit more meaningful key?

Reply:
It means common. I don't want to lengthen it, because you just need to write it over and over. Although we can add this to the docs